docs/SemanticARC.md

```diff
@@ -128,8 +145,41 @@ In order to simplify this, we will make the following changes:
    * @unowned @safe
    * @forwarding
 
+[EE] We really need a good definition of all these conventions here.
+I strongly believe that the local/global notation would help a lot for a good formal definition of the conventions.
+It's hard for the compiler to reason anything about global RCs. But local RCs are easily statically computable and for me, semantic ARC is all about dealing with local RCs.
```

Comment:
As I told you on the chat, I'm not proposing to use the local/global notion in the SIL syntax. I'm just saying it would be useful to _define_ the conventions.
Also, my definition of local/global means that _every_ pairing, which is explicitly expressed in SIL, is local. E.g. store_strong just "converts" from a local to a global RC, i.e. it decrements the local and increments the global RC.